src/authorizations/components/redesigned/ResourceAccordion.tsx

```diff
@@ -0,0 +1,154 @@
+// Libraries
+import React, {Component} from 'react'
+
+// Clockface
+import {Accordion} from '@influxdata/clockface'
+import {ResourceAccordionHeader} from './ResourceAccordionHeader'
+import GetResources from 'src/resources/components/GetResources'
+import {ResourceAccordionBody} from './ResourceAccordionBody'
+
+// Types
+import {AppState, Telegraf, ResourceType} from 'src/types'
+
+// Selectors
+import {getAll} from 'src/resources/selectors'
+import {connect} from 'react-redux'
+import {Bucket} from 'src/client'
+
+export type ReadWritePermissions = {
+  read: boolean
+  write: boolean
+}
+export type SublevelPermissions = {
+  [x: string]: ReadWritePermissions
+}
+
+interface OwnProps {
+  resources: string[]
+}
+interface State {
+  permissions: any
+}
+interface StateProps {
+  telegrafPermissions: any
+  bucketPermissions: any
```

Comment:
do we have any tickets to address the `any` types here? should we?

Reply:
Yes, we should. I created types for these

Reply:
Actually, I am going to create a type for this in a different ticket. This is a bit more complicated, and kind of depends on rest of the resources to be implemented.